packages/block-library/src/image/image.js

```diff
@@ -726,6 +778,10 @@ export default function Image( {
 		);
 	}
 
+	if ( ! url && ! temporaryURL ) {
+		return sizeControls;
+	}
```

Comment:
It feels a little weird to me that the placeholder controls are handles by this conditional instead of being colocated with the `MediaPlaceholder`.

That being said, the Image block structure is already quite a mess, and putting this where I think it should be would require a fair bit of refactoring that probably isn't worth it in this PR.

Reply:
Yes that's what I avoided :D